opentracing-api/src/main/java/io/opentracing/Span.java

```diff
@@ -41,6 +41,9 @@
      */
     void finish();
 
+    // XXX: comment
+    boolean isFinished();
```

Comment:
this is necessary if we want to avoid Tracer/Span wrappers (which I kinda want to avoid)